### DISCUSSION TITLE
legacy MySQL server support

### DISCUSSION THREAD

**Discussion by konopnikV** [mysql-net/MySqlConnector#1167] [Q&A] [answered]
Hi, what is the latest version of MySQL server working with MySqlConnector on .NET6.0? (Looking for solution for MySQL 4.0) Thanks!

**Comment by bgrainger** [comment, marked as answer]
Regular integration tests run with MySQL Server 5.6, and AFAIK MySqlConnector also works with 5.5.

There was a new protocol introduced in MySQL 4.1; MySqlConnector was designed against this new protocol. Thus, it seems highly unlikely it would work with MySQL 4.0.

**Comment by bgrainger** [comment]
![MySQL Connector Versions and MySQL Server Versions](https://user-images.githubusercontent.com/188129/165966433-bacbb6bf-d464-49cb-932a-a0392f38b6f0.png)

From the table above, it looks like you'd need to find a copy of Connector/NET (i.e., MySql.Data) 1.0 to support MySQL Server 4.0. But it seems unlikely that that would work under .NET 6.
